sidecar/config/flags.go

```diff
@@ -25,6 +25,7 @@ const (
 	configFlag              = "config"
 	registerFlag            = "register"
 	proxyFlag               = "proxy"
+	proxyAdapterFlag        = "proxy_adapter"
```

Comment:
Since renaming a flag/env var is a breaking change, I opened an issue for this so we can submit an additional PR for this :
https://github.com/amalgam8/amalgam8/issues/446